test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -293,7 +320,8 @@ private static void runTest9() throws Exception {
         hitKey(KeyEvent.VK_DOWN);
```

Comment:
`addActionListener`, as well as `removeActionListener`, should rather be called on EDT.